docroot/themes/custom/mass_theme/templates/paragraphs/paragraph--social-media.html.twig

```diff
@@ -45,20 +45,23 @@
   * fetchs the social links from the containing node social field.
   */
 #}
-{% embed "@organisms/by-author/social-links-bar.twig" with {
-  "socialLinksBar": {
-    "level": level
-  }
-} %}
-   {% block sectionContent %}
-     {{ social_links }}
-   {% endblock %}
-{% endembed %}
 
-{% if content.field_component_add_separator|field_value|render == "True" %}
-  {% include "@atoms/divider.twig" with {
-    "divider": {
-      "thin": true
+{% if social_links|length > 17 %}
```

Comment:
Why it should be > 17 ?  Is this number correct?

Reply:
That's correct.  `social_links` is not just an array with link content.  It also contains 17 sets of #data along with the link content items [0, 1, ...].   `social_links|length` returns the total number of [17 #data + total content item count].  At the top of the template, it has a note saying "fetchs the social links from the containing node social field."